bundles/org.openhab.binding.kvv/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -8,18 +8,14 @@
 		<description>The KVV bridge connects to the KVV API.</description>
 
 		<config-description>
-			<parameter name="maxTrains" type="integer" required="true" min="0" max="127">
+			<parameter name="maxTrains" type="integer" required="true" min="1" max="127">
 				<label>Maximum Trains Listed</label>
 				<description>Maximum number of trains listed.</description>
 			</parameter>
 			<parameter name="updateInterval" type="integer" required="true" min="1">
```

Comment:
Why not removing the "required" attribute and defining a default value set to 10 which is the default value you consider in the class ?

Reply:
Also, good point.

Reply:
You have to add `<default>10</default>`

Reply:
I did it.